src/components/SpecialResolution/InstructionalText.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <article class="instructional-text section-container">
+  <!-- Hide if correction filing -->
+  <article
+    v-if="isSpecialResolutionFiling"
```

Comment:
Now the comment on line 2 is wrong 🙃

Reply:
I purposely wrote that comment

Reply:
I like the comment, and it's not actually wrong, but I think it would be better to say "only show for ...".